docs/sources/lids/0001-Introduction.md

```diff
@@ -12,7 +12,7 @@ title: "0001: Introducing LIDs"
 
 **Type:** Process
 
-**Status:** In Discussion
+**Status:** Accepted
```

Comment:
Nitpicking: Accepting the LID 0001 should not be part of proposing another LID. Would be great if we could keep that in a separate PR (I know it's overhead ...)

Reply:
Fair point :slightly_smiling_face: I was being lazy.